### PR TITLE
Cawooda/finish routing and seed

### PR DESCRIPTION
**Pull request by cawooda**
Added fairly complete seed data. 

Begun serving objects from model. Before continuing with any more API serving, we need to get a better handle on the views we are designing.

### CODE REVIEW

File: models/index.js
Comment (simonjbr) [inline models/index.js:1]: For the as: user_A and user_B ,the A and B should be lowercase

File: controller/api/postRoutes.js
Comment (simonjbr) [inline controller/api/postRoutes.js:1]: The res.status(500).json(body) should be sending json(error) instead

File: models/index.js
Comment (cawooda) [reply models/index.js:1]: Thanks Simon,
I think them becoming lowercase was Tri's work.

He was showing me how we can let Sequelize handle the creation and management of foreign keys, which is why there is some commenting out in the model definition.

It's working better than it was and can now send back and object with all its associated entities where before we were only getting back the model and a single associated entity.

Comment (simonjbr) [reply models/index.js:1]: Ah I see now it's all commented out in models/Chat.js.

I think we should still make it all lowercase though. Thoughts?

File: seed/userData.json
Comment (simonjbr) [inline seed/userData.json:1]: Looks like a closing double-quote and curly brace are missing from the end of this file

**Review by simonjbr — APPROVED** [review]
Okay I think it all looks fine

**Pull request merged**